plugins/systemstat.koplugin/main.lua

```diff
@@ -25,18 +25,17 @@ function SystemStat:onCharging()
 end
 
 function SystemStat:onNotCharging()
-    self.decharging_count = self.decharging_count + 1
+    self.discharging_count = self.discharging_count + 1
 end
 
 function SystemStat:onCallback()
     local kv_pairs = {
-        {_("Startup at"), os.date("%c", self.start_sec)},
-        {_("Running hours"),
-         string.format("%.2f", os.difftime(os.time(), self.start_sec) / 60 / 60)},
-        {_("Wakeup times"), self.wakeup_count},
-        {_("Sleep times"), self.sleep_count},
-        {_("Charging times"), self.charging_count},
-        {_("Decharging times"), self.decharging_count},
+        {_("KOReader Started at"), os.date("%c", self.start_sec)},
+        {_("Up hours"), string.format("%.2f", os.difftime(os.time(), self.start_sec) / 60 / 60)},
+        {_("Number of wake-ups"), self.wakeup_count},
+        {_("Number of sleeps"), self.sleep_count},
+        {_("Number of chargings"), self.charging_count},
```

Comment:
`Number of charges` (and discharges) would be better than this gerund. Alternatively `Number of charging cycles` would be acceptable, but even then I'd say `Number of charge cycles` is preferable.[1]

Also, I'd personally call the variables `charge_count` and `discharge_count`, but that's a total nitpick.

[1] Apple uses that, for example.
>All rechargeable batteries have a limited number of charge cycles and may eventually need to be serviced. 
Source: http://www.apple.com/batteries/